processor/awsapplicationsignalsprocessor/config/resolvers.go

```diff
@@ -28,6 +28,12 @@ func NewEKSResolver(name string) Resolver {
 	}
 }
 
+func NewEKSResolverWithoutName() Resolver {
+	return Resolver{
+		Platform: PlatformEKS,
+	}
+}
+
```

Comment:
Q: Is it necessary to have this new resolver if we have removed the edit check at https://github.com/amazon-contributing/opentelemetry-collector-contrib/pull/250/files#diff-49c913e4552f87a5361093496c3ebb67f46809b7bd08eee64fd347ca4a0961caR57-R58

Reply:
`NewEKSResolverWithoutName` isn’t actually used in OTel Collector code, they are just used for testing in factory_test.go, config_test.go and resolvers_test.go files.

I'll remove it because it is redundant.